utils/ansible/ftest/templates/daos-launch.sh.j2

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+set -o pipefail
+
+VERSION=0.1
+CWD="$(realpath $(dirname $0))"
+
+DAOS_INSTALL_DIR="{{ daos_runtime_dir }}/install"
+DAOS_SOURCE_DIR="{{ daos_source_dir }}"
+{% if daos_ofi_interface is defined %}
+OFI_INTERFACE="{{ daos_ofi_interface }}"
+{% endif %}
+SERVERS_LIST="{{ groups["daos_servers"] | join(',') }}"
+{% if "daos_clients" in groups and groups["daos_clients"] | length > 0 %}
+CLIENTS_LIST="{{ groups["daos_clients"] | join(',') }}"
+{% endif %}
```

Comment:
(lint) You need a space after the '{'. [SC1054]
(lint) This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]